kolibri/utils/cli.py

```diff
@@ -354,7 +354,7 @@ def restart():
     """
     Restarts the server if it is running
     """
-    if server.restart():
+    if server.restart_and_wait():
```

Comment:
What is it that has changed that makes the "`_and_wait`" and important addition?

Reply:
The `restart` function now just signals that the server should restart - previously it was doing `restart_and_wait` but it was useful to be able to _just_ signal that the server should restart (via the API call) rather than wait for the restart to happen - which always meant that the request failed to respond, because the request thread gets killed while the request is being served.